tutorials/03_plugins.md

```diff
@@ -119,5 +119,5 @@ Ignition GUI will look for display plugins on the following paths, in this
 order:
 
 1. All paths set on the `IGN_GUI_DISPLAY_PLUGIN_PATH` environment variable
-1. `~/.ignition/gui/display_plugins`
-1. [Display plugins that are installed with Ignition GUI](https://ignitionrobotics.org/api/gui/0.1/namespaceignition_1_1gui_1_1display_plugins.html)
+2. `~/.ignition/gui/display_plugins`
+3. [Display plugins that are installed with Ignition GUI](https://ignitionrobotics.org/api/gui/0.1/namespaceignition_1_1gui_1_1display_plugins.html)
```

Comment:
Can you also update the numbers under `### Publisher`?
https://github.com/ignitionrobotics/ign-gui/blame/d9c78efce068db6c85cfa37d3fcadfae9ac29a24/tutorials/03_plugins.md#L51-L56